satpy/readers/native_msg.py

```diff
@@ -62,6 +62,9 @@ class NativeMSGFileHandler(BaseFileHandler):
     """
 
     def __init__(self, filename, filename_info, filetype_info):
+        super(NativeMSGFileHandler,self).__init__(filename, 
+	                                         filename_info, 
+	                                         filetype_info)
         """Initialize the reader."""
```

Comment:
Could put back the docstring just under the function signature ? That's more pythonic